docs/reference/form_types.rst

```diff
@@ -116,6 +116,53 @@ btn_add, btn_list, btn_delete and btn_catalogue:
     (``sonata-admin-setup-list-modal`` by default and
     ``sonata-admin-append-form-element`` when using ``edit:inline``).
 
+sonata_type_model_list
```

Comment:
Please use the FQCN here, even if other fields don't. Using that notation is deprecated or should be.